### PR TITLE
feature/US15607

### PR DESCRIPTION
**Pull request by hcrocker**
- add hidden and locked flags to develop branch

### CODE REVIEW

File: agility.api/src/main/resources/schema/assetProperty.xsd [hunk 1]
```diff
@@ -89,6 +89,26 @@ REPRODUCTION OF WORKS NOT EXPRESSLY AUTHORIZED BY SERVICEMESH IS STRICTLY PROHIB
...
+                  </annox:annotate>
+               </xsd:appinfo>
+            </xsd:annotation>
+         </xsd:element>
+         
+         <xsd:element name="locked" type="xsd:boolean" default="false">
+            <xsd:annotation>
+               <xsd:appinfo>
+                  <annox:annotate target="field">
+                     <a:ApiField comment="True if the property is to be locked and not allow changes." />
```
Comment (Behnke19) [inline agility.api/src/main/resources/schema/assetProperty.xsd]: Doesn't setting writable to false already prevent changes?

Comment (hcrocker) [reply agility.api/src/main/resources/schema/assetProperty.xsd]: this is for HPOO...a field may be writable but deployed, they want to lock it...I saw some checks were isWritable() is used to determine hidden, i.e. if not writable, it is marked as hidden...if it is writable, I apply the value for hidden

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://ci.servicemesh.com:8080/job/csc-agility-platform-sdk-prb/186/

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://ci.servicemesh.com:8080/job/csc-agility-platform-sdk-prb/187/

**Pull request merged**